CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.9.0 (2020-01-22)
+
+- Migrated `skyux upgrade` to [`@skyux-sdk/cli`](https://github.com/blackbaud/skyux-sdk-cli).
```

Comment:
Should we link to a PR? I'd have assumed we would link to https://github.com/blackbaud/skyux-sdk-cli/pull/35.

Reply:
I'm indifferent.  If you think it'd be helpful to consumers, feel free to make the change.

Reply:
Thinking about this more, did you mean link to the PR in this repo?  I think linking to the cli repo is sufficient, but I am going to add the PR in this repo for the change I made (like we normally do, but I failed to do).